src/main/java/org/openhab/binding/zwave/handler/ZWaveControllerHandler.java

```diff
@@ -562,14 +554,13 @@ public void ZWaveIncomingEvent(ZWaveEvent event) {
                     break;
                 case IncludeFail:
                     eventKey = ZWaveBindingConstants.EVENT_INCLUSION_FAILED;
-                    eventState = BindingEventType.WARNING;
```

Comment:
Why change this? I'd prefer to leave it as is.

Reply:
I thought it would be more consistent because `SecureIncludeFailed` uses `BindingEventType.ERROR` as well. 
I will revert it - it doesn't relate to the pull request, so it shouldn't be done here

Reply:
Ok - sounds fine.